### PR TITLE
Split content in categories

### PR DESCRIPTION
**Pull request by amandahla**
I had the same feeling as Gregory mentioned in [this comment](https://github.com/canonical/is-charms-contributing-guide/pull/62#issuecomment-1508155168) so this is a content categorization suggestion to make the guide more readable. Reviews are welcome :)

### CODE REVIEW

**Review by arturo-seijas — DISMISSED** [review]
Nice! IMO :)

File: README.md [hunk 1]
```diff
@@ -2,11 +2,21 @@
...
+This project outlines expectations the IS Charms
 team has of each other when contributing code to any of the repositories owned
 by the team. Changes can be proposed by anyone through pull requests (including
 to anything in this file).
 
+Categories:
+
+* [Contributions](CONTRIBUTIONS.md): How to contribute to Juju charms,
+how to set repository, how to review a PR etc.
+* [Definitions](DEFINITIONS.md): Descriptions of some terms in this guide.
```
[the remark this answers is not in the document]
Comment (amandahla) [reply README.md:14]: Usually, I create a list in alphabetical order. Do you think would be better to change to a different order?

File: README.md [hunk 2]
```diff
@@ -2,11 +2,21 @@
...
+* [Juju Charms](JUJUCHARMS.md): Juju Charms version control, configuration,
+libraries etc.
+* [Python](PYTHON.md): What to check before submitting a PR.
```
[the remark this answers is not in the document]
Comment (amandahla) [reply README.md:17]: I didn't notice that, good catch! Rephrased.

**Comment by jdkandersson** [comment]
This should be on hold for now given we're working on merging this with the main charm style guide, we can re-assess the contributing guide after that is done. In the meantime, it is still applicable as it has been

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nrobinaubertin) [reply README.md:14]: I was suggesting that because definitions should maybe be read first

**Review by merkata — COMMENTED** [review]
I like this a lot! I think we still might want to keep a lightweight CONTRIBUTING.md file around, as it is a best practice to have one (people expect there to be one and they could open it up first imo) and it links to further documents. The structure is great otherwise.

**Comment by merkata** [comment]
> This should be on hold for now given we're working on merging this with the main charm style guide, we can re-assess the contributing guide after that is done. In the meantime, it is still applicable as it has been

Fair point, however splitting it up in advance might also provide some benefit with content being more structured and organized, that has to be considered as well. Merging might necessitate splitting up the content either way due to the volume of information.

**Pull request closed without merging**